README.md

```diff
@@ -1 +1,42 @@
-# OOP-school-library
\ No newline at end of file
+# OOP-school-library
+
+A Ruby library to manage a school library using latest technologies.
+
+## Built With
+
+- Major languages: Ruby
+- Technologies used: Git, Github and Gitflow,
+
+## Getting Started
+
+### Setup
+- git clone https://github.com/juanmanuelbyc/OOP-school-library.git
+
+## Authors
```

Comment:
- [ ] OPTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor.